legacy/builder/test/hardware/arduino/avr/boards.txt

```diff
@@ -1,18 +0,0 @@
-
```

Comment:
I'm struggling to understand the original code. Am I correct that sole purpose of [the `legacy/builder/test/hardware/arduino` folder](https://github.com/arduino/arduino-cli/tree/scerza/fix-legacy-platforms-loading/legacy/builder/test/hardware/arduino) is a place to store the test data files `boards.local.txt` and `platform.local.txt`, from which they will be copied into the downloaded `arduino:avr` platform for use in the `TestLoadHardware` test?
https://github.com/arduino/arduino-cli/blob/0ee486d622ba9a73d7d2b507e0383f4750bc6360/legacy/builder/test/hardware_loader_test.go#L32-L33

Reply:
Yes, that's correct. 

It used to be loaded by the hardware loader, `boards.local.txt` and `platform.local.txt` overwrote `boards.txt` and `platform.txt` of an unversioned platform with same packager and architecture. That was a mistake though since it's not how it must work, both the `local.txt` file must live in the same folder of their "sibling" to work correctly.

It used to work just because manually installed platforms don't have a version set, with the changes I made #1166 I set the version of manually installed platforms so this tests all broke.

Reply:
To make things less confusing, my suggestion is to move these files to a folder that doesn't look like a boards platform. When I see a folder named something like `hardware/arduino/avr`, especially when that `hardware` folder is used as an Arduino hardware folder, I expect that to contain a platform, so it's confusing to me to discover that this `hardware/arduino/avr` is simply a place to store some test data for use elsewhere and is never used as a platform.

Reply:
Yeah, makes sense. I see how it could be confusing, I'll change it.

Reply:
Thanks!